### DISCUSSION TITLE
Package manager

### DISCUSSION THREAD

**Discussion by chenzhekl** [odin-lang/Odin#1479] [Ideas/Requests]
Will there be an official package manager for Odin just like what Cargo is for Rust?

**Comment by hstormo** [comment]
Just gonna quote bill here:
> There are zero plans to have an official package manager for Odin because of numerous reason. One reason is that package managers are trivial piece of software IF AND ONLY IF the concept of a package is well defined within the language itself. Another is that I am not a huge fan of the culture that package managers help to facilitate. I recommend this interview I was in: https://www.youtube.com/watch?v=2YLA4ajby00

↳ https://discord.com/channels/568138951836172421/585077163250941962/983722992708255765

**Reply by sumofat** [reply]
Nothing stopping there from being an unofficial one though

**Reply by MrBns** [reply]
ohh no.. thats weird.
